### PR TITLE
Cleans up some martial art code.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

title.
Also renames the NOGUNS trait to CHUNKYFINGERS because that more closely represents what it does. Readds NOGUNS and actually makes it prevent the user from using guns. Also gives the CHUNKYFINGERS trait to hulk and removes some snowflake checks.

## Why It's Good For The Game

This is overall cleaner code, that alone makes this good for the game.

## Changelog
:cl:
code: Martial Art and NOGUN cleanup.
/:cl:

### CODE REVIEW

File: code/__DEFINES/traits.dm [hunk 1]
```diff
@@ -68,6 +68,7 @@
 #define TRAIT_HUSK				"husk"
 #define TRAIT_BADDNA			"baddna"
 #define TRAIT_CLUMSY			"clumsy"
+#define TRAIT_CHUNKYFINGERS		"chunckyfingers" //means that you can't use weapons with normal trigger guards.
```
Comment (tralezab) [inline code/__DEFINES/traits.dm]: "Chuncky" might wanna correct that

Comment (nemvar) [reply code/__DEFINES/traits.dm]: Eh. It's more unique that way and prevents possible conflicts :tada:

**Comment by Dawson1917** [comment]
I like em big
I like em chunky

**Comment by actioninja** [comment]
conflict

**Comment by nemvar** [comment]
Again? Fucks sake.

**Comment by nemvar** [comment]
shut up travis

**Pull request merged**